nodejs-apps/node-entry-point/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:12-alpine
+ARG MANAGER_IP
```

Comment:
Consider changing this to `IAST_MANAGER_IP` (since one of the Java flows uses `IAST_MANAGER_IP` environment variable).

Reply:
Done